pkg/deploy/dashboard/dashboard.go

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright (c) 2012-2019 Red Hat, Inc.
```

Comment:
Minor 2012-2019 -> 2021

Reply:
I just copied pasted it from the existing files.
I would prefer not to touch it here not to retrigger all the tests again.

but in general it should be solved with automation, like devworkspace operator has https://github.com/devfile/devworkspace-operator/blob/main/Makefile#L242

Reply:
fixed